applications/ALEapplication/python_scripts/mesh_solver_base.py

```diff
@@ -3,6 +3,9 @@
 import KratosMultiphysics
 import KratosMultiphysics.ALEApplication as KratosALE
 KratosMultiphysics.CheckForPreviousImport()
+import mesh_solver_base
```

Comment:
Is this file importing itself?

Reply:
It is, can you please revert the changes in this file?

Reply:
Note that you should not change the existing files of other applications (such as ALE one here) until unless it is completely necessary. Is mesh_solver_base a standard file (the existing one) or is it added by you?

Reply:
It a standard file, 

sure will do dears,